bootstrap-salt.sh

```diff
@@ -4185,6 +4194,43 @@ install_debian_check_services() {
 #   Fedora Install Functions
 #
 
+__install_saltstack_fedora_onedir_repository() {
+    if [ "$ITYPE" = "stable" ]; then
+        repo_rev="$ONEDIR_REV"
+    else
+        repo_rev="latest"
+    fi
+
+    __PY_VERSION_REPO="yum"
+    if [ -n "$_PY_EXE" ] && [ "$_PY_MAJOR_VERSION" -eq 3 ]; then
+        __PY_VERSION_REPO="py3"
+    fi
+
+    GPG_KEY="SALT-PROJECT-GPG-PUBKEY-2023.pub"
+
+    REPO_FILE="/etc/yum.repos.d/salt.repo"
+
+    if [ ! -s "$REPO_FILE" ] || [ "$_FORCE_OVERWRITE" -eq $BS_TRUE ]; then
+        fetch_url="${HTTP_VAL}://${_REPO_URL}/${_ONEDIR_DIR}/${__PY_VERSION_REPO}/fedora/${DISTRO_MAJOR_VERSION}/${CPU_ARCH_L}/${ONEDIR_REV}"
+        if [ "${ONEDIR_REV}" = "nightly" ] ; then
+            fetch_url="${HTTP_VAL}://${_REPO_URL}/${_ONEDIR_NIGHTLY_DIR}/${__PY_VERSION_REPO}/fedora/${DISTRO_MAJOR_VERSION}/${CPU_ARCH_L}/"
+        fi
+
+        __fetch_url "${REPO_FILE}" "${fetch_url}.repo"
+
+        for key in $GPG_KEY; do
```

Comment:
`$GPG_KEY` can be an array or a list of strings?
Why do we itereate?

Reply:
I copied both the Photon and Fedora functions from the CentOS ones, which at one point had support for multiple keys.  I've updated both to only use the one.